examples/advanced-filter-convert-publish/res/configuration.toml

```diff
@@ -2,19 +2,19 @@
 LogLevel = 'INFO'
     [Writable.StoreAndForward]
     Enabled = false
-    RetryInterval = 50000 # 5mins
+    RetryInterval = '300s'
```

Comment:
Use `"5m"` here in in all other config files

Reply:
ok